src/main/scala/org/deepdive/ddlog/DeepDiveLogCompiler.scala

```diff
@@ -550,8 +550,11 @@ class QueryCompiler(cq : ConjunctiveQuery, hackFrom: List[String] = Nil, hackWhe
             nonCategoryWeightCols += s"${deepdiveWeightColumnPrefix}${i}"
         }
 
+        val valueExpr = if (stmt.valueExpr != null) stmt.valueExpr else IntConst(1)
```

Comment:
Maybe this is non-critical and better kept simple/uniform, but I wish there was a nice way to pass the default value = 1 implicitly rather than repeating the same constant everywhere down to the grounded binaries for every factor.